src/app/help/help-routing.module.ts

```diff
@@ -0,0 +1,26 @@
+import { NgModule } from '@angular/core';
+
+/** Routing Imports */
+import { Route } from '../core/route/route.service';
+import { Routes, RouterModule } from '@angular/router';
+import { extract } from '../core/i18n/i18n.service';
+
+/** Translation Imports */
+import { HelpComponent } from './help.component';
+
+const routes: Routes = [
+    Route.withShell([
+        {
+            path: 'help',
+            component: HelpComponent,
+            data: { title: extract('Help'), breadcrumb: 'Help' }
+        }
```

Comment:
This helpComponent has the UI which shows all the shortcuts, so sould i rename it to keyboard-Shorcuts Component because in community app it was the help page.

Reply:
You may rename it and create a separate help component or keep this and create a keyboard-shortcuts component and update accordingly as I have specified above.

Reply:
Just asking, what will the help component contain as we'll be doing the help routing to the API docs, so what will be there in the help component?

Reply:
The implementation will be similar to the self-service component as I have mentioned above. Please have a look at it.